benchmarks/100.webapps/120.uploader/python/function.py

```
@@ -8,17 +8,21 @@
 def handler(event):
-  
+
     bucket = event.get('bucket').get('bucket')
     output_prefix = event.get('bucket').get('output')
     url = event.get('object').get('url')
     name = os.path.basename(url)
     download_path = '/tmp/{}'.format(name)
 
     process_begin = datetime.datetime.now()
-    urllib.request.urlretrieve(url, filename=download_path)
+    req = urllib.request.Request(url)
```

Ruff (0.14.0)

Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)